src/components/LoginTest.jsx

```diff
@@ -1,13 +1,54 @@
 import React, { useState } from 'react';
-import { logInWithEmailAndPassword } from '../utils/auth_utils';
+import { instanceOf } from 'prop-types';
+import { cookieConfig, cookieKeys, Cookies, withCookies } from '../utils/cookie_utils';
+import {
+  auth,
+  logInWithEmailAndPassword,
+  useNavigate,
+  signInWithGoogle,
+} from '../utils/auth_utils';
 
-const LoginTest = () => {
+const LoginTest = ({ cookies }) => {
+  const navigate = useNavigate();
   const [email, setEmail] = useState();
   const [password, setPassword] = useState();
 
-  const handleSubmit = e => {
+  /**
+   * This function handles logging in with email/password (standard log in)
+   * If the user signs in successfully, they are redirected to /logout, otherwise they are redirected to the login screen
+   * @param {Event} e
+   */
+  const handleSubmit = async e => {
     e.preventDefault();
-    logInWithEmailAndPassword(email, password);
+    const success = await logInWithEmailAndPassword(email, password);
+    if (success) {
+      navigate('/logout');
+      cookies.set(cookieKeys.ACCESS_TOKEN, auth.currentUser.accessToken, cookieConfig);
+    } else {
+      navigate('/');
+    }
```

Comment:
This logic should be moved into the `logInWithEmailAndPassword` function in `auth_utils` so its easier for NPO teams to implement. The redirect path should be a prop passed into the function for the same reason. 
Also add a try/catch around the `logInWithEmailAndPassword` in the case an error was thrown inside of it. In the catch, the error message will be put in the `errorMessage` state which will then be outputted in the JSX.